### ISSUE TITLE
Customer Segment Feasibility

### ISSUE THREAD

**Issue by daairola** [oats-group3/General-Work#13]
I'm struggling to understand why you would want to target wealthier demographics to sell products to. It would likely be much more profitable to market items towards less wealthy individuals because of how much of a larger demographic they make up of the population. You could still try to find quality products for them but instead find the best cost effective items.

**Comment by genesiscartagena** [comment]
I agree. The wealthier are sometimes demanding and won't buy a product if it doesn't surpass their expectations. Also, the wealthier class is less populated than the rest of the classes, so I believe business will be slower. Lower classes don't ask for much as long as the product is effective and they can use it for what they need.
